lib/byebug/printers/texts/base.yml

```diff
@@ -16,6 +16,7 @@ break:
     not_changed: "Incorrect expression \"{expr}\", breakpoint not changed"
   confirmations:
     delete_all: "Delete all breakpoints?"
+  deleted: "Delete breakpoint {pos}"
```

Comment:
I think you added the new message here, but are still using the hardcoded string in the code.